public/js/createEntry.js

```diff
@@ -1,23 +1,27 @@
 const entryForm = async (e) => {
+    
     e.preventDefault();
-
+   
     const title = document.querySelector('#title').value.trim();
     const content = document.querySelector('#content').value.trim();
     const background = document.querySelector('#background-color').value;
+    const url = window.location.href.split('/');
+    const id = url[4];
+
 
 
     if(title && content && background){
-        const response = await fetch('/api/entry', {
+        const response = await fetch('/api/entries', {
             method: 'POST',
-            body: JSON.stringify({title, content, background}),
+            body: JSON.stringify({title, content, background, id}),
             headers: {'Content-Type': 'application/json'},
         });
         if(response.ok){
-            document.location.replace('/journal');
+            document.location.replace(`../../journals/${id}`);
         } else {
             alert('Failed to create entry');
         }
     }
 };
 
-document.querySelector('#newEntry').addEventListener('submit', entryForm);
```

Comment:
This is actually the correct version.

Reply:
that newEntry id doesn't exist

Reply:
Nevermind. I just checked and there are two IDs. The one you're referring to that's on the form and the one I used here that's on the button of the form. I just removed the form id so there's no more confusion